CMakeLists.txt

```diff
@@ -130,11 +130,6 @@ install(FILES
             postinstall.txt
         DESTINATION doc/turnserver
             COMPONENT Runtime)
-install(FILES examples/etc/turnserver.conf
```

Comment:
Why is this being removed?

If it needs to be removed, it should be done as a standalone commit.